src/hotspot/share/opto/vectorization.hpp

```diff
@@ -445,6 +451,39 @@ class VLoopTypes : public StackObj {
   const Type* container_type(Node* n) const;
 };
 
+// Submodule of VLoopAnalyzer.
+// We compute and cache the VPointer for every load and store.
+class VLoopPointers : public StackObj {
```

Comment:
Nit: Should we call this `VLoopVPointers` to make the link to `VPointers` and not just some pointers?

Reply:
Hmm. This looks like a general naming question now. My idea is that the `V` at the beginning of the types just is kind of a "namespace", to say that all types are used for `Vectorization`.
But I guess here we can do it just so everybody knows we are dealing with `VPointers`. I'll make an exception, but don't want to see `V`'s littered everywhere ;)

Reply:
I agree that we should not start adding `V`'s in between. But for `VPointer`, I think it makes sense since `VPointer` is quite a known name and `LoopPointers` might be misleading. Thanks for making this change!